src/components/inlineEdit/package.json

```diff
@@ -15,7 +15,7 @@
     "dependencies": {
         "jquery": "3.6.0",
         "jquery-ui": "1.12.1",
-        "fluid-framework": "4.0.0",
+        "fluid-framework": "^4.0.0",
```

Comment:
Is there a reason to change the version type of `fluid-framework` from "4.0.0" to "^4.0.0"?

Reply:
> Is there a reason to change the version type of `fluid-framework` from "4.0.0" to "^4.0.0"?

After reading the code below, seems the reason is to have the installer fetch the latest 4.1.0 if it's available. This makes sense at the dev stage. Shall we take a note when cutting a new infusion version,  all dependencies still use the same exact version rule?

Reply:
No, I don't think so. We are moving to a model where the framework will be released and versioned separately from the components, and this version update is meant to communicate that the framework has been bumped one version number but the components haven't. The caret is meant to leave room for further such updates - e.g. that inlineEdit version 4.0.0 is expected to be compatible with any framework version 4.x

Reply:
I see. Thanks.